### ISSUE TITLE
error "No path specified! Can not get relative" when compiling

### ISSUE THREAD

**Issue by KutnerBitsrc** [teambit/bit#2133]
I'm trying to compile a component, and getting this error:
```No path specified! Can not get relative.```

Always happens for compiler react-typescript@3.0.4
3.0.4 was my 'go to' version because it managed to actually build, including .d.ts, as long as I limited the component to a single file 😢

It is especially bad because I have a lot of component using this compiler that **I can't tag and export back**. 😵
react-typescript@3.0.35 obviously do not work and tries to compile node_modules. 😭

I believe I tagged components with this compiler last week (~6.11.19), so maybe it's a breaking change in an indirect dependency? 

I tried the following bit versions: 14.4.3, 14.4.1, 14.2.5, 14.2.0, 14.1.3.

bit version   : 14.4.3
node version  : v12.12.0
npm version   : 6.11.3
yarn version  : 1.19.1
platform      : darwin

**Comment by GiladShoham** [comment]
@KutnerUri please provide steps to reproduce

**Comment by KutnerBitsrc** [comment]
1. ```bit import amit.capsule-dev/xterm/capsule-addon```
2. edit
3. ```bit build```
--> error

**Comment by davidfirst** [comment]
@KutnerBitsrc , If I remember correctly, this exact error was fixed once you updated the compiler to the latest typescript. The react-typescript is deprecated.
When I debugged this error on another component you had, I noticed that the react-typescript compiler returns an array of files and `undefined` values. These undefined values are the ones that cause Bit to generate the error above.
cc: @GiladShoham , @qballer maybe explicitly setting react-typescript as deprecated could help. (if not done already).
